components/layouts/ThePageFooter/FooterSection.vue

```diff
@@ -1,13 +1,21 @@
 <template>
   <section class="footer-section">
-    <h2 class="footer-section__title">
+    <h2
+      class="footer-section__title"
+      :class="{'footer-section__title_dark-mode': darkMode }"
```

Comment:
can we combine these class attributes to be combined into one attribute?

Reply:
I thought there was another way of having the conditional in one line... but if this is the convention we're going with, that's good for me to know.